OpenRA.Game/FileFormats/Png.cs

```diff
@@ -404,9 +417,9 @@ public byte[] Save()
 			}
 		}
 
-		public void Save(string path)
+		public void Save(string path, bool fast = false)
```

Comment:
Are there any cases where we actually care more about being small rather than being fast?

Reply:
we do use this for saving map previews, which then get uploaded to the resource centre

Reply:
I'd be interested to see how much of a difference this actually makes (i suspect it may be negligible).

Reply:
I ran d2k sheets with #21719

| Slow   | Fast   |
| ------ | ------ |
| 148 KB | 203 KB |
| 1,1 MB | 1,4 MB |
| 1,2 MB | 1,5 MB |
| 1,8 MB | 2,3 MB |
| 1,9 MB | 2,4 MB |
| 16 KB  | 18 KB  |
| 18 KB  | 26 KB  |
| 40 KB  | 71 KB  |

very significant differences

Reply:
For map previews?

Reply:
no, for sheets. For maps you'd have to script map preview regeneration and test. I know the dotnet provided best compression isn't even that good #20911

Reply:
My point was that if the increase in size of map previews is negligible that we should use `BEST_SPEED` a;waus/

Reply:
I tend to lean on the side that we want the best possible compression. Though I dunno if maps using zip ruins that. Adding compression ontop of compression can have the opposite effect